pkgs/build-support/setup-hooks/patch-shebangs.sh

```diff
@@ -19,9 +19,8 @@ patchShebangs() {
     local newInterpreterLine
 
     find "$dir" -type f -perm -0100 | while read f; do
-        if [ "$(head -1 "$f" | head -c+2)" != '#!' ]; then
-            # missing shebang => not a script
-            continue
+        if ! isScript "$f"; then
```

Comment:
Also seems faster the the original due less forking.

Reply:
@aszlig any further remarks?

Reply:
No forking at all actually.

Reply:
Although I personally find `isScript "$f" || continue` more readable.